crates/db_views/src/comment/comment_view.rs

```diff
@@ -85,6 +88,7 @@ impl CommentView {
       .left_join(person_actions_join)
       .left_join(instance_actions_join)
       .left_join(comment_creator_community_actions_join)
+      .left_join(local_user_join)
```

Comment:
These join to the local_user table to be able to get the admin status of `my_person`